app/controllers/application_controller.rb

```diff
@@ -22,17 +22,17 @@ def record_security_event(event, options = {})
 
 protected
 
-  def get_payload(payload = nil)
-    if payload
+  def get_payload(payload_string = nil)
```

Comment:
You could do something like this to avoid having to set `nil` initially:
```
    jwt = if payload_string
            jwt = Jwt.create!(
              jwt_payload: payload_string,
            )
            session[:jwt_id] = jwt.id
            jwt
          elsif session[:jwt_id]
            Jwt.find(session[:jwt_id])
    end
    jwt&.jwt_payload&.deep_symbolize_keys
```

Reply:
I've done a similar thing in e1a5c8c - does that look better to you?

Reply:
Looks great!